open-build-service/notqmail-rpmlintrc

```diff
@@ -0,0 +1,14 @@
+addFilter(".* unversioned-explicit-self-provides")
```

Comment:
Is this file also OpenSUSE Enterprise only?

Reply:
No. This is needed in case we provide a virtual library let's say qmail-libs and apt-get install qmail-libs should install libqmail.so.x.y in the system lib directory. We don't provide any shared libraries. This has been removed in the latest commit